src/Core/Grid/Presenter/AccessbilityChecker/Row/EmployeeDeleteAccessibilityChecker.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Grid\Presenter\AccessbilityChecker\Row;
```

Comment:
looking at the whole class, what we need to implement is not that but a dedicated Symfony voter: don't reinvent the wheel => https://symfony.com/doc/3.4/security/voters.html, or in French => https://openclassrooms.com/fr/courses/5489656-construisez-un-site-web-a-l-aide-du-framework-symfony-4/5654131-securisez-lacces-de-votre-site-web#/id/r-5749915 :sunglasses:

Reply:
I had this idea but wanted to keep `Core` as much framework agnostic as possible, but I guess it's not worth. :+1:

Reply:
@matks wdyt about using Symfony Voters? I think it's a good idea, but I don't know how exactly it should look like as it's highly connected to security/permissions component. :thinking: